src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -648,11 +648,11 @@ int UProbeManager::DeployOpenSSLUProbes(const absl::flat_hash_set<md::UPID>& pid
       // before the BPF map is updated. This value is cleaned up when the upid is
```

Comment:
This comment will be out of date I think.

Is it possible to break this up (granted this might just be a future PR though)... i.e. like so:
1. Check if static ssl uprobes are appropriate.
2. Attach.
This is different from just "TryAttach" which is what we have now. The above two step approach would enable us to update the map before attach. (But I'm not sure how important that update before attach is, i.e. the out-of-date comment here makes me think it is important to update then attach).

Reply:
ah, good call. I went ahead and removed that comment. I believe we don't need to do it optimistically and that's something I'll clean up with the following refactor.